build_projects/dotnet-host-build/build.ps1

```diff
@@ -8,7 +8,7 @@ param(
     [string]$Architecture="x64",
     [string]$TargetArch="",
     [string]$ToolsetDir="",
-    [string]$Framework="netcoreapp1.0",
+    [string]$Framework="netcoreapp1.1",
```

Comment:
The corresponding file to change is https://github.com/dotnet/core-setup/blob/f758ae62fba9b65ed2796f4ba04c49d696683c33/build_projects/dotnet-host-build/build.sh#L122